tests/test_resource.py

```diff
@@ -370,7 +371,10 @@ def test_cors_headers_param_request(self) -> None:
             ALLOW_ORIGIN_HEADER: ORIGIN_EXAMPLE,
         }
 
-    @patch.dict(environ, {"CORS_HEADERS": "X-PRN-KEY,X-PRN-TOKEN"})
+    @patch(
+        "lbz.resource.CORS_HEADERS",
+        MockedConfig("x", parser=lambda _a: ["X-PRN-KEY", "X-PRN-TOKEN"]),
```

Comment:
I have a feeling that we need to add the possibility to clear configuration 🤔

Each test should be treated more or less separately, otherwise, workarounds like this one will be added randomly in the code - unfortunately, this is the wrong approach.